core/src/main/scala/cats/Eval.scala

```diff
@@ -310,6 +310,30 @@ private[cats] trait EvalInstances extends EvalInstances0 {
       }
     }
 
+  implicit val catsReducibleForEval: Reducible[Eval] =
```

Comment:
Hmm sorry for all of the thinking out loud here, but I just realized that the `traverse` implementation above only calls `.value` once, just as the `Reducible` instances do. Is there a reason that we would consider this to be different? Maybe because the `Eval` in the return type suggests that we aren't calling `.value`?